### ISSUE TITLE
mimetypelist.js fails integrity check after `occ maintenance:mimetype:update-js`

### ISSUE THREAD

**Issue by Alphakilo** [nextcloud/server#16259]
## Steps to reproduce
1. Run `occ maintenance:mimetype:update-js`
2. Check the integrity (`occ integrity:check-core`)

### Expected behaviour
No integrity check failures.

### Actual behaviour
Integrity check fails:
```sh
# occ integrity:check-core
  - INVALID_HASH:
    - core/js/mimetypelist.js:
      - expected: dc48de7ad4baa030c5e563350c9a80b274bad783f6f5adbf1595ecef6c6a32e52890a24cb26cddb0aa20193ba52c001150c68d8bfb567f0aed566f4029a190a3
      - current: 53613657e01ad07fb878200f174a72a8ac062f6f3d52624034c85157a2bd7517b6b79b18ddad5b312319e7d9bec6b1d409c8ec9098480a356777e0dcee2f39f8
```

Diffset to 16.0.2 "baseline":
```sh
# diff -u /usr/local/www/apache24/data/nextcloud/core/js/mimetypelist.js /tmp/server-16.0.2/core/js/mimetypelist.js
--- /usr/local/www/apache24/data/nextcloud/core/js/mimetypelist.js      2019-07-05 18:55:32.629124000 +0200
+++ /tmp/server-16.0.2/core/js/mimetypelist.js  2019-07-04 08:47:21.000000000 +0200
@@ -100,7 +100,8 @@
     "text/x-ldif": "text/code",
     "text/x-python": "text/code",
     "text/x-shellscript": "text/code",
-    "web": "text/code"
+    "web": "text/code",
+    "application/internet-shortcut": "link"
 },
        files: [
     "application",
```

### Server configuration

**Operating system**: HARDENEDBSD-12-STABLE

**Web server:** Apache/2.4.39 (FreeBSD)

**Database:** PostgreSQL 11.3 on amd64-portbld-freebsd12.0

**PHP version:** PHP 7.3.5 (cli) (built: May 12 2019 17:37:02) ( NTS )

**Nextcloud version:** 16.0.2

**Updated from an older Nextcloud/ownCloud or fresh install:** Updated from 16.0.1 via WebUI, ran `occ upgrade` as the last step.

**Where did you install Nextcloud from:** nextcloud/server/releases/tag/v16.0.0.tar.gz

**Signing status:**
<details>
<summary>Signing status</summary>

```
Technical information
=====================
The following list covers which files have failed the integrity check. Please read
the previous linked documentation to learn more about the errors and how to fix
them.

Results
=======
- core
	- INVALID_HASH
		- core/js/mimetypelist.js

Raw output
==========
Array
(
    [core] => Array
        (
            [INVALID_HASH] => Array
                (
                    [core/js/mimetypelist.js] => Array
                        (
                            [expected] => dc48de7ad4baa030c5e563350c9a80b274bad783f6f5adbf1595ecef6c6a32e52890a24cb26cddb0aa20193ba52c001150c68d8bfb567f0aed566f4029a190a3
                            [current] => 53613657e01ad07fb878200f174a72a8ac062f6f3d52624034c85157a2bd7517b6b79b18ddad5b312319e7d9bec6b1d409c8ec9098480a356777e0dcee2f39f8
                        )

                )

        )

)
```
</details>

**List of activated apps:**
<details>
<summary>App list</summary>

```
Enabled:
  - accessibility: 1.2.0
  - activity: 2.9.1
  - admin_audit: 1.6.0
  - bookmarks: 1.0.6
  - bruteforcesettings: 1.3.0
  - calendar: 1.7.0
  - checksum: 0.4.3
  - cloud_federation_api: 0.2.0
  - comments: 1.6.0
  - contacts: 3.1.3
  - dav: 1.9.2
  - federatedfilesharing: 1.6.0
  - files: 1.11.0
  - files_accesscontrol: 1.6.0
  - files_automatedtagging: 1.6.0
  - files_downloadactivity: 1.5.0
  - files_markdown: 2.0.6
  - files_pdfviewer: 1.5.0
  - files_retention: 1.5.1
  - files_rightclick: 0.13.0
  - files_sharing: 1.8.0
  - files_texteditor: 2.8.0
  - files_trashbin: 1.6.0
  - files_versions: 1.9.0
  - files_videoplayer: 1.5.0
  - firstrunwizard: 2.5.0
  - gallery: 18.3.0
  - logreader: 2.1.0
  - lookup_server_connector: 1.4.0
  - metadata: 0.9.0
  - nextcloud_announcements: 1.5.0
  - notifications: 2.4.1
  - oauth2: 1.4.2
  - password_policy: 1.6.0
  - previewgenerator: 2.1.0
  - privacy: 1.0.0
  - provisioning_api: 1.6.0
  - quota_warning: 1.5.0
  - recommendations: 0.4.0
  - serverinfo: 1.6.0
  - sharebymail: 1.6.0
  - support: 1.0.0
  - survey_client: 1.4.0
  - systemtags: 1.6.0
  - tasks: 0.11.0
  - text: 1.0.0
  - theming: 1.7.0
  - twofactor_backupcodes: 1.5.0
  - twofactor_nextcloud_notification: 1.1.1
  - twofactor_totp: 2.1.2
  - twofactor_u2f: 3.0.0
  - updatenotification: 1.6.0
  - viewer: 1.0.0
  - workflowengine: 1.6.0
Disabled:
  - encryption
  - federation
  - files_external
  - files_snapshots
  - impersonate
  - user_ldap
```
</details>

**Nextcloud configuration:**
<details>
<summary>Config report</summary>

```
{
    "system": {
        "instanceid": "***REMOVED SENSITIVE VALUE***",
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "secret": "***REMOVED SENSITIVE VALUE***",
        "trusted_domains": [
            "files.mydomain.tld"
        ],
        "trusted_proxies": "***REMOVED SENSITIVE VALUE***",
        "datadirectory": "***REMOVED SENSITIVE VALUE***",
        "dbtype": "pgsql",
        "version": "16.0.2.1",
        "overwrite.cli.url": "http:\/\/files.mydomain.tld",
        "dbname": "***REMOVED SENSITIVE VALUE***",
        "dbhost": "***REMOVED SENSITIVE VALUE***",
        "dbport": "",
        "dbtableprefix": "oc_",
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "installed": true,
        "memcache.local": "\\OC\\Memcache\\APCu",
        "memcache.locking": "\\OC\\Memcache\\Redis",
        "redis": {
            "host": "***REMOVED SENSITIVE VALUE***",
            "port": 0,
            "dbindex": 0,
            "password": "***REMOVED SENSITIVE VALUE***",
            "timeout": 1.5
        },
        "mail_smtpmode": "smtp",
        "mail_smtpsecure": "tls",
        "mail_sendmailmode": "smtp",
        "mail_from_address": "***REMOVED SENSITIVE VALUE***",
        "mail_domain": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpauth": 1,
        "mail_smtpauthtype": "PLAIN",
        "mail_smtphost": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpport": "587",
        "mail_smtpname": "***REMOVED SENSITIVE VALUE***",
        "mail_smtppassword": "***REMOVED SENSITIVE VALUE***",
        "versions_retention_obligation": "30, auto",
        "maintenance": false,
        "loglevel": 2,
        "theme": "",
        "updater.release.channel": "stable",
        "updater.secret": "***REMOVED SENSITIVE VALUE***"
    }
}
```
</details>

**Comment by Alphakilo** [comment]
Somewhat duplicates #10411, might be solved with #15810 but I'm still curious as to why running the `occ` command would alter the file from the "known good" / default state.

**Comment by kesselb** [comment]
Can we close this as duplicate of #10411?
